reference/6/Microsoft.PowerShell.Utility/ConvertFrom-Json.md

```diff
@@ -127,6 +136,25 @@ Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
+### -AsHashtable
+Converts the JSON to a hash table object. This switch was introduced in PowerShell 6.0.
+There are several scenarios where it can overcome some limitations of the `ConvertFrom-Json` cmdlet
+- If the JSON contains a list with keys that only differ in casing. Without the switch, those keys would be seen as identical keys and therefore only the last one would get used.
+- If the JSON contains a key that is an empty string. Without the switch, the cmdlet would throw an error since a `PSCustomObject` does not allow for that but a hash table does. An example use case where this can occurs are `project.lock.json` files.
+- Hash tables can be processed faster for certain data structures.
+ 
+ ```yaml
+ Type: SwitchParameter
+ Parameter Sets: UNNAMED_PARAMETER_SET_1
```

Comment:
Since this switch is not part of a parameter set, I think you just delete this line